dwitter/feed/views.py

```diff
@@ -83,6 +83,7 @@ def feed(request, page_nr, sort):
                'header_title': 'Dwitter',
                'feed_type': 'all',
                'page_nr': page,
+               'on_last_page': last == dweet_count,
```

Comment:
[Lines 61 and 62](https://github.com/tbroadley/dwitter/blob/b489a4fa509f74fd0c2638a5647acbccd0d174bf/dwitter/feed/views.py#L61-L62):

```py
if (last >= dweet_count):
    last = dweet_count
```

So if `last` was 20 and `dweet_count` was 15, `last` will be set to 15. Then we check if `last == dweet_count`, which is true.